lisa/sut_orchestrator/hyperv/schema.py

```diff
@@ -0,0 +1,37 @@
+# Copyright (c) Microsoft Corporation.
+# Licensed under the MIT license.
+
+from dataclasses import dataclass, field
+from typing import List
+
+from dataclasses_json import dataclass_json
+
+
+@dataclass_json
+@dataclass
+class HypervServer:
+    address: str
+    username: str
+    password: str
+
+
+@dataclass_json
+@dataclass
+class ExtraArgs:
+    command: str
+    args: str
+
+
+@dataclass_json
+@dataclass
+class HypervPlatformSchema:
+    servers: List[HypervServer] = field(default_factory=list)
+    extra_args: List[ExtraArgs] = field(default_factory=list)
+
+
+@dataclass_json
+@dataclass
+class HypervNodeSchema:
+    hyperv_generation: int = 2
+    vhd: str = ""
```

Comment:
In Azure platform, we're moving to an unified image source object, called AzureImageSchema. Please add an `ImageSchema` in root `schema.py`, and inherit a `VHDImageSchema` in sut_orchestrator folder. So later, when Azure is refactoring, it can use this class.

Reply:
I don't have the full context on this. What should be the fields in `ImageSchema` and `VHDImageSchema`?

Reply:
The `ImageSchema` can be empty, and we will add more fields later like `cpu_arch`, `data_disk`. There may be another layer called `AzureImageSchema`, add data_path, vhd_gen. For `VhdAzureImageSchema`, it needs another file to support CVM testing. Check related classed in Azure to see how they are used.

Reply:
Please see the latest change after introducing VHDSource. As of now, we don't need any ImageSchema. We can add later on a need basis when we want to introduce metadata fields. No point in introducing empty & unused classes.

Reply:
Source/build is related to copy files. ImageSchema is to describe image metadata. The metadata is used to decide VM settings, when create vm, or a test case should be picked or not. We have some settings already, cpu arch, vhd gen, cvm... The empty class is to reduce cost of this change, it doesn't mean not needed.

Reply:
Added.